ui/admin/app/components/form/field/list-wrapper/select/index.js

```diff
@@ -0,0 +1,59 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { set } from '@ember/object';
+
+export default class MappingListSelectComponent extends Component {
+  // =attributes
+
+  /**
+   * @type {string}
+   */
+  @tracked newOptionValue = '';
+
+  get options() {
```

Comment:
Using a getter here to retrieve the nested property within the `model` object by using the `name` of the select-field.

Reply:
I think you still to take in an `@options` and can't assume the consumer will use a `model`.

Reply:
I was saying we should still keep the getter but check for `@options` so a consumer can provide that or a model